README.md

```diff
@@ -10,115 +10,62 @@ This repo is meant as a template which you can use to create a Discord bot very
 
 - [Deno](https://deno.land)
 
-## Step By Step
+## Optional
+- [Docker](https://docs.docker.com)
```

Comment:
I would add a 1 liner next to this to explain what its doing. WHy should a user use this if its optional